gulpfile.js

```diff
@@ -97,6 +97,11 @@ const traverseJson = (jsonTree, descriptionCallback, prefixPath) => {
     }
 };
 
+// remove invalid characters that aren't allowed in xlf.
+const removeInvalidXLFCharacters = (str) => {
+    return str.replace(/(<)|(>)/g, "");
```

Comment:
Can you replace `(<)|(>)` with `<|>`? I don't understand why you put them in a capture group that doesn't seem to be used (unless I'm misunderstanding something).

Reply:
Yes, I originally thought that I needed it but just tested and your suggestion is correct. Pushed the changes.